src/components/ArticlePage.jsx

```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+
+const ArticlePage = () => {
+  const [getArticle, setGetArticle] = useState([]);
+  const { article_id } = useParams();
+
+  useEffect(() => {
+    axios
+      .get(`https://news-v9aq.onrender.com/api/articles/${article_id}`)
+      .then((response) => {
+        setGetArticle(response.data.article);
+      });
+  }, [article_id]);
+
```

Comment:
Having a separate api file for all your request functions could make this cleaner.